models.go

```diff
@@ -555,7 +555,33 @@ func (x reqExternalContactMarkTag) intoBody() ([]byte, error) {
 	return result, nil
 }
 
-// reqExternalContactMarkTag 编辑企业客户标签
+// respExternalContactMarkTag 编辑企业客户标签
 type respExternalContactMarkTag struct {
 	respCommon
 }
+
+// reqJSCode2Session 临时登录凭证校验
+type reqJSCode2Session struct {
+	JSCode string
+}
+
+var _ urlValuer = reqJSCode2Session{}
+
+func (x reqJSCode2Session) intoURLValues() url.Values {
+	return url.Values{
+		"js_code":    {x.JSCode},
+		"grant_type": {"authorization_code"},
+	}
+}
+
+// respJSCode2Session 临时登录凭证校验
+type respJSCode2Session struct {
+	respCommon
+	JSCode2Session
+}
+
+type JSCode2Session struct {
```

Comment:
`JSCodeSession` might be better naming, the `to` part makes the type name too verb-like instead of model-like IMO.

The method name `JSCode2Session` unfortunately must remain for it to remain easily accessible by casual searches.

Reply:
Also add godoc comments for the whole type, as it is public API.